web/src/narrow.js

```diff
@@ -359,6 +359,9 @@ export function activate(raw_terms, opts) {
         // recursively.
         reset_ui_state();
 
+        // We sometimes prevent draft restoring until the narrow resets.
+        compose_state.allow_draft_restoring();
```

Comment:
This could probably reasonably go in `reset_ui_state`... As could `compose_banner.clear_message_sent_banners();`, now that look at it.

Reply:
Great point, moved it and added a commit for moving `clear_message_sent_banners`